### ISSUE TITLE
Multi arch builds

### ISSUE THREAD

**Issue by mnorrsken** [tiangolo/uvicorn-gunicorn-docker#47]
I've created multi-arch builds over at https://hub.docker.com/r/mrnr91/uvicorn-gunicorn-fastapi
(for arm64 arm/v7 amd64)

However I'd rather have them over here of course :) Is there a particular reason that you don't have multi arch builds?

(Note - python-slim wasnt buildable probably because pip had to build packages from source)

**Comment by stickperson** [comment]
@mnorrsken do you have the source code? I assume this is a bit more involved than changing `FROM python:3.8` to `FROM arm64v8/python:3.8`

**Comment by mnorrsken** [comment]
Not really. Its just that you have to build the dependency images with multi arch too. Tbh I moved on from this and now I just use a standard python image with fastapi and uvicorn in requirements.

**Comment by tiangolo** [comment]
Thank you!

Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-docker#-warning-you-probably-dont-need-this-docker-image

And by using an official Python base image you would get multi-arch by default. :sunglasses:
